### DISCUSSION TITLE
Updating srcpkgs

### DISCUSSION THREAD

**Discussion by ACR-Jeff** [void-linux/void-packages#51125] [Q&A] [answered]
I have looked through tons of documentation and I can't find any info on Updating srcpkgs, What I mean by that is I noticed on the issue bug report that there is a script to update srcpkgs, Say I have easyeffects as a srcpkg.
If I give the command 
`./xbps-src update-check easyeffects` 
the output is:
```
easyeffects-7.1.5 -> easyeffects-7.1.6
easyeffects-7.1.5 -> easyeffects-7.1.7
```
In it's directory it also has a file named "update"
with the contents:
```
site="https://github.com/wwmm/easyeffects/tags"
pattern="releases/tag/v\K[0-9.]+"
```
Also if there is such a way to update can this be done when srcpkg isn't installed prior to building?

Any information would be helpful, Thank you

**Comment by classabbyamp** [comment, marked as answer]
see [this section of the manual](https://github.com/void-linux/void-packages/blob/master/Manual.md#updates)

`update-check` is only there to notify package maintainers of new releases being available, you have to edit the template to update it to that version yourself (update `version`, update the checksums, fix any build issues, etc)

**Reply by ACR-Jeff** [reply]
> see [this section of the manual](https://github.com/void-linux/void-packages/blob/master/Manual.md#updates)
> 
> `update-check` is only there to notify package maintainers of new releases being available, you have to edit the template to update it to that version yourself (update `version`, update the checksums, fix any build issues, etc)

Thank you for a quick response, You've helped me a ton, Now time to start digging in to learn more. :)

**Comment by meator** [comment]
https://xbps-src-tutorials.github.io/package-update-tutorial.html
